tower-http/src/services/fs/serve_dir/future.rs

```diff
@@ -215,7 +215,10 @@ fn build_response(output: FileOpened) -> Response<ResponseBody> {
         .header(header::ACCEPT_RANGES, "bytes");
 
     if let Some(encoding) = output.maybe_encoding {
```

Comment:
(You will need to add `PartialEq` as well to the enum for this to work)
```suggestion
if let Some(encoding) = output
        .maybe_encoding
        .filter(|encoding| encoding != Encoding::Identity)
```